web/src/components/vis/MetaboliteTable.vue

```diff
@@ -22,6 +22,9 @@ export default {
   data() {
     return {
       format: format('.4e'),
+      pagination: {
+        rowsPerPage: -1,
+      },
```

Comment:
Does this just mean "as many pages as possible"?

Reply:
https://v15.vuetifyjs.com/en/components/data-tables

```js
{
  descending: boolean",
  page: number",
  rowsPerPage: number // -1 for All",
  sortBy: string",
  totalItems: number"
}
```

Reply:
A comment in our code might be a good idea.